srv/src/main/java/my/bookshop/handlers/CatalogServiceHandler.java

```diff
@@ -58,15 +59,15 @@
 class CatalogServiceHandler implements EventHandler {
 
 	private final PersistenceService db;
-	private final DraftService reviewService;
+	private final ReviewService.Draft reviewService;
 
 	private final Messages messages;
 	private final FeatureTogglesInfo featureToggles;
 	private final RatingCalculator ratingCalculator;
 	private final CqnAnalyzer analyzer;
 
-	CatalogServiceHandler(PersistenceService db, @Qualifier(ReviewService_.CDS_NAME) DraftService reviewService,
-			Messages messages, FeatureTogglesInfo featureToggles, RatingCalculator ratingCalculator, CdsModel model) {
+	CatalogServiceHandler(PersistenceService db, ReviewService.Draft reviewService, Messages messages,
```

Comment:
I think this class doesn't actually require the `DraftService` APIs.

```suggestion
	CatalogServiceHandler(PersistenceService db, ReviewService reviewService, Messages messages,
```

Reply:
changed